terraform/modules/bulk_upload/bucket.tf

```diff
@@ -20,6 +20,32 @@ resource "aws_s3_bucket_public_access_block" "this" {
   restrict_public_buckets = true
 }
 
+
```

Comment:
The policy has been duplicated here across the two files, but it seemed better than creating a module with just a bucket policy. I think if we end up duplicating it more than twice we should modularise